backend/pick-git/build.gradle

```
@@ -27,6 +27,12 @@ dependencies {
 
 	asciidoctor 'org.springframework.restdocs:spring-restdocs-asciidoctor:1.2.6.RELEASE'
 
+	compileOnly 'org.projectlombok:lombok:1.18.20'
+	annotationProcessor 'org.projectlombok:lombok:1.18.20'
+
+	testCompileOnly 'org.projectlombok:lombok:1.18.20'
+	testAnnotationProcessor 'org.projectlombok:lombok:1.18.20'
+
 	testImplementation('org.springframework.restdocs:spring-restdocs-mockmvc')
 	testImplementation 'org.springframework.boot:spring-boot-starter-test'
 	testImplementation group: 'io.rest-assured', name: 'rest-assured', version: '4.4.0'
```

```suggestion
	testImplementation 'io.rest-assured:rest-assured:4.4.0'
```

여기도 이렇게 형식 통일하면 좋을 거 같습니다 !